CHANGELOG-PENDING.md

```diff
@@ -7,6 +7,7 @@ Month, DD, YYYY
 ### BREAKING CHANGES
 
 - [go package] (Link to PR) Description @username
+[feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
```

Comment:
What is `cel-shed`?

Reply:
Oh, I see: 
![image](https://user-images.githubusercontent.com/7441766/145438286-205f617c-be9d-4f6e-95cb-4749088d4f80.png)